rucapenlabs/leyyer-landing-page
# Create sign in page

## Added
- Router
- New Sign In page
- Components for new Sign In page
- Icons for Sign In page
- Connection to the backend app
- ESLint and Prettier

## Changed
- Removed old Sign In modal
- Fixed all lint errors and formatted with prettier

## Testing
1. Run the backend app and add a new user
2. Clone this branch
3. `npm run dev`
4. Click Sign In -> Should relocate to /signin
5. Enter credential -> Should relocate to the app.leyyer.com if succeded
Check also unsuccessful state
